src/components/button.jsx

```diff
@@ -8,12 +8,13 @@ const Button = (props) => {
       type={props.type ?? "button"}
       disabled={props.disabled}
       class={`${classNames(styles.btn, {
-        [styles.primary]: props.category === "primary",
-        [styles.secondrary]: props.category === "secondrary",
-        [styles.tertiary]: props.category === "tertiary",
-        [styles.tertiary_light]: props.category === "teriary_light",
-        [styles.flat]: props.category === "flat",
-        [styles.trade]: props.category === "trade",
+        [styles.primary]: props.type === "primary",
```

Comment:
why is this change? @likhith-deriv please help check

Reply:
@sanjam-deriv  can u please change these back to category. type prop of button is already binded with type attribute of button. Hence this will not work on buttons